include/woff2/output.h

```diff
@@ -12,6 +12,7 @@
 #include <algorithm>
 #include <cstring>
 #include <memory>
+#include <stdint.h>
```

Comment:
It is better to follow
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes